.github/workflows/ci.yml

```diff
@@ -57,19 +68,30 @@ jobs:
         run: |
           composer update --no-interaction --prefer-dist --ansi --no-progress
 
+      - name: Wait for selenium to start
+        run: |
+          curl --head -X GET --silent --show-error --retry 60 --retry-connrefused --retry-delay 1 --max-time 10 http://172.18.0.2:4444
```

Comment:
what is this hardcoded IP here ?

Reply:
It's the same one as `DRIVER_URL` a few lines below.

It references docker on the host machine (we are running selenium in docker while running tests on the host machine).
Normally, one would use 127.0.0.1/localhost or even 0.0.0.0, but it doesn't seem to work in github.
When I have some time, I'll see if I can come up with an alternative.

Reply:
is this IP guaranteed to stay the same ?

Otherwise, it might be better to make docker expose the port on localhost

Reply:
btw, looking at the docker-compose file, it looks like you _already_ expose the port.

Reply:
> btw, looking at the docker-compose file, it looks like you already expose the port.

Yes, that's what I meant by

> Normally, one would use 127.0.0.1/localhost or even 0.0.0.0, but it doesn't seem to work in github.

Somehow github doesn't bind the port to localhost, but it's accessible from that ip I used.

Anyway, I'll try a few approaches here: https://github.com/uuf6429/webdriver-classic-driver/pull/1
If I can't manage, I'll at least try to find some reference on how I got that ip and add it as a comment.

Reply:
Fixed!